examples/analyse_counterpoise.py

```diff
@@ -4,51 +4,51 @@
 TODO: Needs high level description
 
 '''
-def test_analyse_counterpoise():
 
+
+def test_analyse_counterpoise():
     from os import getcwd, chdir
     from ase.io import read
 
     from carmm.analyse.counterpoise_onepot import counterpoise_calc
     from carmm.run.aims_calculator import get_aims_calculator
     from carmm.run.aims_path import set_aims_command
-    from carmm.utils.python_env_check import ase_env_check
-
 
     # This is an example script for using counterpoise_calc for counterpoise (CP) correction. Please note the species
     # files in data/CO_BSSE are fake ones and default species settings are also deleted from aims.out.
 
+    set_aims_command(hpc='hawk', basis_set='light', defaults=2020)
     CO = read('data/CO_BSSE/C_monoxide_pbe.traj')
     examples_directory = getcwd()
 
-#    # This is now all managed inside get_aims_calculator, so not needed here
-#    # Keeping in case needed for legacy purposes
-#    # Old calculator:
-#    # Construct the calculator
-#
-#    if not ase_env_check('3.23.0'):
-#        toy_calc = get_aims_calculator(dimensions=0, xc='pbe', default_initial_moment=0.5,
-#                                       spin='collinear'
-#                                       directory=examples_directory+'/data/CO_BSSE',
-#                                       species_dir=examples_directory+'/data/CO_BSSE')
-#        toy_calc.set(spin='collinear', relativistic='atomic_zora scalar')
-#    else:
-#        from ase.calculators.aims import AimsProfile, Aims
-#        fake_profile = AimsProfile(command='', default_species_directory=examples_directory + '/data/CO_BSSE')
-#        toy_calc = get_aims_calculator(dimensions=0, xc='pbe', spin='collinear',
-#                                       default_initial_moment=0.5,
-#                                       directory=examples_directory+'/data/CO_BSSE',
-#                                       profile=fake_profile,
-#                                       species_dir=examples_directory+'/data/CO_BSSE')
-#        # Reverted this so we still use the get_aims_calculator - means we use one process only for definition
-#        #toy_calc = Aims(xc='pbe', spin='collinear', default_initial_moment=0.5,
-#        #                               relativistic='atomic_zora scalar', directory=examples_directory+'/data/CO_BSSE',
-#        #                               species_dir=examples_directory+'/data/CO_BSSE', profile=fake_profile)
+    #    # This is now all managed inside get_aims_calculator, so not needed here
```

Comment:
The legacy code can be retrieved from version history, I suggest that this should be removed.